docs/samples/contact-center/app.js

```diff
@@ -157,10 +181,15 @@ function register() {
             idleCodesDropdown.add(option);
           }
         });
-
     }).catch((error) => {
         console.error('Event subscription failed', error);
     })
+
+    webex.cc.eventEmitter.on('task:incoming', (task) => {
```

Comment:
Shouldn't this be `webex.cc.on('task:incoming', () => {});`. eventEmitter doesn't seem right.